dir.traversal.targets

```diff
@@ -78,7 +78,7 @@
       </Project>
 
       <!-- Include all projects which have TestTFM from the supported set -->
-      <ProjectsToTest Include="@(Project)" Condition="'%(Project.Extension)'=='.csproj' And $([System.String]::new('%(Project.TestTFMs)').Contains(';$(FilterToTestTFM);'))">
+      <ProjectsToTest Include="@(Project)" Condition="'%(Project.Extension)'=='.csproj' And $([System.String]::new('%(Project.Identity)').Contains('.Tests'))">
```

Comment:
I think we will still need some sort of filtering in here, as there are some verticals where we don't want to run some test projects.